packages/subgraph/src/helpers/loaders.ts

```diff
@@ -49,6 +49,7 @@ export function loadMarketById(id: string): MarketPlace {
     marketPlace.durationAverage = BigInt.zero();
     marketPlace._durationTotal = BigInt.zero();
     marketPlace.totalNumberOfLenders = BigInt.zero();
+    marketPlace.paymentType = "";
```

Comment:
we didn't always have this value when creating a market - we should default the value to `EMI`
```suggestion
    marketPlace.paymentType = "EMI";
```